scripts/eqt_env.yml

```diff
@@ -5,5 +5,5 @@ channels:
 dependencies:
   - python
   - pip
-  - pyside2
```

Comment:
The user will need to remember to install a Qt binding package. Do we want that?